examples/dpu.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: config.openshift.io/v1
+kind: DpuOperatorConfig
+metadata:
+  name: dpu-operator-config
+  namespace: dpu-operator-system
```

Comment:
Q1) is this the namespace we will be using for the operator?

Q2) we probably want to have a yaml to create the aforementioned namespace.

Reply:
Q1) short term yes

Q2) in the future, I agree.